src/frontc/cabs2cil.ml

```diff
@@ -439,7 +437,6 @@ let newAlphaName (globalscope: bool) (* The name should have global scope *)
     findEnclosingFun !scopes;
   let newname, oldloc =
            AL.newAlphaName ~alphaTable:alphaTable ~undolist:None ~lookupname:lookupname ~data:!currentLoc in
-  H.add varnameMapping origname newname;
```

Comment:
Hopefully you know this better than I do, but it wasn't obvious to me whether `varnameMapping` is entirely contained within `environment` or not, because they're added to in somewhat different places. I remember seeing somewher that `environment` was only conditionally added to.

Reply:
It does not add temporary values introduced by CIL, but searching for their use is not supported anyway iirc.